### ISSUE TITLE
SphereGeometry missing many parameters

### ISSUE THREAD

**Issue by iliyang** [jovyan/pythreejs#104]
I like how **pythreejs** provides a very thin wrapper around **three.js**. This, I think, facilitates maintenance.

The `SphereGeometry` class in **three.js** has a bunch of parameters:
```javascript
SphereGeometry(radius, widthSegments, heightSegments, phiStart, phiLength, thetaStart, thetaLength)
```
but the `SphereGeometry` class in **pythreejs** only supports the `radius` param. That's a pity because the sphere tessellation precision cannot be controlled (done via the `widthSegments` and `heightSegments` params in **three.js**).

Is there a plan to add such support? And is there a list of what's supported and what not? Such a list could facilitate others to contribute to keeping **pythreejs** up-to-date.

**Comment by honorabel** [comment]
This is actually the goal of a big initiative w/ the project to autogenerate a lot of the wrappers to match the threejs api much more closely.  It is actively being worked on right now.

So, to answer your question, yes, that's in the plan :)

**Comment by iliyang** [comment]
Yey! I'm absolutely loving **pythreejs** and it is great to hear new and exciting stuff is coming!

**Comment by vidartf** [comment]
This should now be fixed via #88. Please reopen if there's anything I missed.
